addons/mail/tests/test_mail_tools.py

```diff
@@ -167,6 +167,28 @@ def test_mail_find_partner_from_emails_multicompany(self):
                 found = Partner._mail_find_partner_from_emails([self._test_email], records=record)
                 self.assertEqual(found, [expected], f'Found {found[0].name} instead of {expected[0].name}: {msg}')
 
+    def test_mail_find_partner_from_emails_alias_localpart(self):
```

Comment:
Isn't it possible to add an entry in existing tests to avoid a complete new test just for that ?